### DISCUSSION TITLE
Dark Mode on Android does'nt work for the HTML content

### DISCUSSION THREAD

**Discussion by reithi** [tutao/tutanota#3913] [Bugs]
The dark mode for Android does not change the message content. So, for a visually impaired user like me it is not usefull. I need a dark background to see anything on the screen. If a large area is white I can not see the rest of the screen. IMHO a dark mode should change the HTML content as well. K-9 Mail is doing a good job on it. I would appreciate if tutanota would do as well.
Thanks

**Comment by charlag** [comment]
We do change background for emails that don't define their own colors. For complex email which define text color or a background we have to resort to the white background.

Is there any information on how K-9 achieves changing the email content?

**Reply by reithi** [reply]
I haven't looked into it because I am not familiar with the code. Isn't it just overwriting some kind of a content.css file. Sorry for my stupid questions;)
